components/scim/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/impl/SCIMUserManager.java

```diff
@@ -1150,4 +1224,73 @@ private void provisionSCIMOperation(int provisioningMethod, SCIMObject provision
             throw new CharonException("Error in initializing provisioning handler", e);
         }
     }
+
+    private void provision(ProvisioningOperation provisioningMethod, SCIMObject provisioningObject) throws CharonException {
+
+        String domainName = UserCoreUtil.getDomainName(carbonUM.getRealmConfiguration());
+        try {
+
+            Map<org.wso2.carbon.identity.application.common.model.ClaimMapping, List<String>> outboundAttributes = new HashMap<org.wso2.carbon.identity.application.common.model.ClaimMapping, List<String>>();
```

Comment:
This has already fixed with https://github.com/wso2/carbon-identity/commit/c96c8fb63e38406d20a16b776ec387fc78e89342